### DISCUSSION TITLE
SEQ default port

### DISCUSSION THREAD

**Discussion by ruxann** [datalust/seq-tickets#1841] [Help] [answered]
Hello,
I use Seq API keys in my project and want to block Seq with the default port to run in the application.
Is it possible to block Seq with the default port?

I have followed the guide documented here: https://docs.datalust.co/docs/using-serilog

**Comment by nblumhardt** [comment]
Thanks for getting in touch. I'm not 100% certain I understand what you mean by "block" in this context. Could you please share a bit more info?

Many thanks,
Nick

**Reply by ruxann** [reply]
I am using serilog and in the first example of the document it sends a log without an api key to the URL with port 5341. In the example of dynamic level control an API key is used to send a log. How can I limit logging to the server from only applications that have an API key? And block the logging requests from the first example without an API key?

**Reply by nblumhardt** [reply, marked as answer]
Thanks for your reply! In the Seq UI, you can go to Settings > API keys > tick "Require an API key for HTTP/S ingestion".

Let me know if you need any more info.

**Reply by ruxann** [reply]
it is working. Thank you
